images/baremetal/Dockerfile.ci

```diff
@@ -0,0 +1,24 @@
+# This Dockerfile is a used by CI to publish an installer image
+# It builds an image containing only the openshift-install.
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN TAGS="libvirt baremetal" hack/build.sh
+
+
+FROM registry.svc.ci.openshift.org/origin/4.1:base
+COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+
+RUN yum install --setopt=tsflags=nodocs -y \
+    yum update -y && \
```

Comment:
are we supposed to be running `yum update` or just rely on UBI refreshes for this?

Reply:
The other platform images do the same (openstack, libvirt) - we've followed their example here

Reply:
This is at least consistent with other Dockerfiles that install some rpms, see `images/installer/Dockerfile.upi.ci` and `images/openstack/Dockerfile.ci`

Reply:
You never run yum update ever.